apps/escuela-frontend/studio/schemas/objects/resources/explainer.js

```diff
@@ -0,0 +1,72 @@
+import {MdOndemandVideo} from 'react-icons/md'
+
+export default {
+  name: 'explainer',
+  type: 'object',
+  title: 'Explainer',
+  description:
+    'A type of Lesson that can be a sectionIntro moduleIntro moduleOutro',
+  icon: MdOndemandVideo,
+  fields: [
+    {
+      name: 'label',
+      title: 'Label',
+      type: 'string',
+      hidden: true,
+    },
+    {
+      name: 'title',
+      title: 'Title',
+      type: 'string',
+      validation: (Rule) => Rule.max(90),
+    },
+    {
+      name: 'slug',
+      title: 'Slug',
+      type: 'slug',
+      validation: (Rule) => Rule.required(),
+      options: {
+        source: 'title',
+        maxLength: 96,
+      },
+    },
+    {
+      name: 'explainerType',
+      title: 'Explainer Type',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+      options: {
+        list: [
+          {title: 'Explainer to Section', value: 'sectionIntro'},
+          {title: 'Explainer to Module', value: 'moduleIntro'},
+        ],
+      },
+    },
+    {
```

Comment:
How should `explainer` lessons be displayed? The React Tutorial contains three explainer videos at the start of the course: 

1. Tutorial Overview
2. Project setup
3. Example Run-through

This lesson is not an exercise; it's more of a slide-driven lesson. It can be added to a section or a module. I created a Welcome section and inside dropped the explainers. 


![CleanShot 2022-11-09 at 12 41 57@2x](https://user-images.githubusercontent.com/57044804/200937556-d85a8ce2-5f95-4562-9d8f-2bde160b36a1.png)